docs/contributing.rst

```diff
@@ -24,24 +24,22 @@ We welcome contributions from external contributors, and this document describes
    versions <https://github.com/pyenv/pyenv-virtualenv>`_ or :code:`conda` to create virtual environments with `different Python
    versions <https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#managing-environments>`_.
 
-5. Install :code:`poetry` using the following command. :code:`poetry` is a better dependency resolver than :code:`pip`.
+5. Install :code:`poetry`. Instead of installing Poetry with :code:`pip`, follow the official Poetry installation guide using pipx:
```

Comment:
How about:

You will require [poetry](https://python-poetry.org/) to manage the dependencies of :code:`toqito`. Please consult the [installation instructions](https://python-poetry.org/docs/#installation) for further details. 

We don't necessarily want to explicitly mention `pip`, `pipx`, etc., as this is all subject to change from the side of poetry.